src/components/Mousy.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2021 The Aha001 Team.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {useRef} from 'react';
+import {useFrame} from '@react-three/fiber';
+import {useGLTF} from '@react-three/drei';
+
+import {quaternionFrom, getHeadRotation} from '../utils/keypoints';
+import {Vector3} from 'three';
+
+/**
+ * Renders a <Mousy/> component.
+ *
+ * @param {Props} props
+ * @return {string} rendered JSX.
+ */
+export default function Mousy(props) {
+  let kp;
+  const group = useRef();
+  const {nodes, materials} = useGLTF('../../../mousy.glb');
+
+  useFrame((state, delta) => {
+    kp = props.keypoints.current;
+
+    if (typeof kp !== 'undefined' && kp !== null) {
```

Comment:
Optional:

[Google style](https://google.github.io/styleguide/jsguide.html#features-equality-checks-exceptions) prefers `==` and `!=` to check equality where coercion is desirable:

> Catching both null and undefined values:

```
if (someObjectOrPrimitive == null) {
  // Checking for null catches both null and undefined for objects and
  // primitives, but does not catch other falsy values like 0 or the empty
  // string.
}
```

Reply:
Replaced with !=. Thanks.